host_vars/mysql.opencraft.com/public.yml

```diff
@@ -0,0 +1,21 @@
+MYSQL_SERVER_DOMAIN: "mysql.opencraft.com.local"
```

Comment:
Sorry if I used `.com` in other places - the right domain to use would be `mysql.opencraft.hosting`, since that's where we'll be putting hosting resources from now on.